.github/ISSUE_TEMPLATE/03_documentation.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Documentation
+about: Submit an issue on documentation
+title: (Set a clear title describing your idea)
+labels: 'untriaged'
+assignees: ''
+---
+
+(Describe your issue and goal here)
+  
+### The page URLs
+  
+(List the URLs to update here)
+
+* (examples in this repository)
+
+### Requirements
+
+Please make sure if this topic is specific to this SDK. For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:
```

Comment:
Can we reword this a bit?

~Please make sure if this topic is specific to this SDK.~
Please make sure this is a documentation issue related to Emoji Manager.